Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG VERSION
 LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-ARG GOPHISH_VERSION="0.7.1"
+ARG GOPHISH_VERSION="0.11.0-cisa.1"
```

Comment:
Why you no use `+` metadata?  
e.g., `0.11.0+cisa.1`

Reply:
Thanks @felddy for the reminder about this.  I did use `+` instead of `-` originally, but when GitHub Actions created the release assets (e.g. https://github.com/cisagov/gophish/releases/tag/v0.11.0-cisa.1), the download URLs for the platform-specific zip files were different from what was expected.

Instead of generating URLs like this: `https://github.com/cisagov/gophish/releases/download/v0.11.0%2Bcisa.1/gophish-v0.11.0%2Bcisa.1-linux-64bit.zip`

GitHub Actions was generating URLs like this: `https://github.com/cisagov/gophish/releases/download/v0.11.0%2Bcisa.1/gophish-v0.11.0.cisa.1-linux-64bit.zip`

The first `+` was being correctly URL encoded to `%2B`, but the second one was being converted to a `.` and that was going to be a hassle for us here: https://github.com/cisagov/gophish-docker/blob/f21b923ea0b07c7c3ab3f4a6edb487a577a21e5c/Dockerfile#L33-L35